gmso/external/convert_hoomd.py

```diff
@@ -0,0 +1,881 @@
+"""Convert GMSO Topology to GSD snapshot."""
+from __future__ import division
+
+import itertools
+import json
+import re
+import statistics
+import warnings
+
+import hoomd
```

Comment:
We either should add hoomd to the environment file for gmso, or raise an exception when the user is trying to use `convert_hoomd.py` and does not have hoomd in their environment.

Reply:
Added the has_hoomd check